ScoutSuite/providers/gcp/resources/stackdriverlogging/sinks.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+
+from ScoutSuite.providers.base.configs.resources import Resources
+
+class Sinks(Resources):
+    def __init__(self, sdl_facade, project_id):
+        self.sdl_facade = sdl_facade
+        self.project_id = project_id
+
+    async def fetch_all(self):
+        raw_sinks = await self.sdl_facade.get_sinks(self.project_id)
+        for raw_sink in raw_sinks:
+            sink_name, sink = self._parse_sink(raw_sink)
+            self[sink_name] = sink
+
+    def _parse_sink(self, raw_sink):
```

Comment:
I started using `_parse` instead of `_parse_<insertredundantnamehere>` in my other PRs. I don't think there is a case where you would have multiple parse methods in one resource class, unlike with the previous implementation. What do you think?

Reply:
Maybe `_parse` is just a bit too generic, over something like `_parse_resource`?

Reply:
I also prefer `_parse_resouce_name`; I believe it is clearer. Simply `_parse` leads me to think that the method parses all the resources (what the class represents), but it just parses one.

Reply:
> Simply `_parse` leads me to think that the method parses all the resources (what the class represents), but it just parses one at a time.

 Good call, I hadn't thought of that! I liked the consistency of `_parse` and the fact that it was short, but the confusion it might cause outweighs the cleanness.